ppl/cmd/zar/README.md

```diff
@@ -265,54 +278,54 @@ zar ls custom.zng
 ```
 To see what's in it:
 ```
-find $ZAR_ROOT -name custom.zng | head -n 1 | xargs zq -f table 'head 10' -
+find $ZAR_ROOT -name idx-`for file in $ZAR_ROOT/indexdefs/*; do   zq -z $file | grep 'name:"custom.zng"' > /dev/null; if [ $? = 0 ]; then echo $file | sed 's/.*indexdefs\/idxdef-//'; fi done` | head -n 1 | xargs zq -f table 'head 10' -
 ```
 You can see the IPs, counts, and _path strings.
 
 At the bottom you'll also find a record describing the micro-index layout. To
 see it:
 
 ```
-find $ZAR_ROOT -name custom.zng | head -n 1 | xargs zq -f table 'tail 1' -
+find $ZAR_ROOT -name idx-`for file in $ZAR_ROOT/indexdefs/*; do   zq -z $file | grep 'name:"custom.zng"' > /dev/null; if [ $? = 0 ]; then echo $file | sed 's/.*indexdefs\/idxdef-//'; fi done` | head -n 1 | xargs zq -f table 'tail 1' -
 ```
 
 ### zar find with custom index
 
 And now I can go back to my example from before and use "zar find" on the custom
 index:
 ```
-zar find -z -x custom.zng 10.164.94.120 | zq -t -
+zar find -z -x custom.zng 10.164.94.120
 ```
 Now we're talking!  And if you take the results and do a little more math to
 aggregate the aggregations, like this:
 ```
-zar find -z -x custom.zng 10.164.94.120 | zq -f table "count=sum(count) by _path" -
+zar find -x custom.zng 10.164.94.120 | zq -f table "count=sum(count) by _path | sort -r" -
```

Comment:
I intentionally dropped the `-z` here to try and show the best practice of "for best performance, always using binary ZNG when you're piping to downstream `zq`" (but I also didn't wanna get into a big preachy tangent about that either). Part of me wanted to leave the `-z` there just to show me adding to the prior command line and not have the reader wondering if there was significance my dropping it, since it's probably not a lesson that needs to be driven home in this small-scaled emo. A close call in my mind, but probably not a big deal either way.